src/components/contextual/pages/pool/PoolChart.vue

```diff
@@ -240,15 +240,19 @@ function getFeesData(
   isAllTimeSelected: boolean,
   pariodLastSnapshotIdx: number
 ) {
-  const feesValues = periodSnapshots.map(
-    (snapshot, idx): readonly [string, number] => {
+  const feesValues = periodSnapshots
+    .map((snapshot, idx) => {
       const value = parseFloat(snapshot.swapFees);
       let prevValue: number;
 
       // get value of prev snapshot
       // if it is last value among all snapshots, then prev value is 0
       if (idx === snapshotValues.value.length - 1) {
-        prevValue = 0;
+        if (isAllTimeSelected) {
+          return;
```

Comment:
need these changes since we don't have proper last prev value for old pools all time period snapshots data